content/sensu-go/5.0/_index.md

```diff
@@ -11,12 +11,10 @@ layout: "single"
 
 <iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-go&type=star&count=true" frameborder="0" scrolling="0" width="87px" height="20px"></iframe> | [Share your feedback][2]
 
-Sensu is the monitoring event pipeline built to reduce operator burden and make developers and business owners happy.
-Started in 2011, Sensu’s flexible approach solves the challenges of monitoring hybrid-cloud and ephemeral infrastructures with scalable, automated workflows and integrations with tools you already use.
+Sensu is the industry leading solution for multi-cloud monitoring at scale. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their multi-cloud environments. Founded in 2017, Sensu offers a comprehensive monitoring solution for enterprises, providing complete visibility across every system, every protocol, every time — from Kubernetes to bare metal.
```

Comment:
I'm hesitant to describe Sensu as a comprehensive solution... I feel like this is in conflict with the idea that Sensu is a comprehensive monitoring _framework_ which is used with other tools to build a solution.

Reply:
cc @bananabele

Reply:
@cwjohnston @apaskulin that's a great point, though from a corporate and product marketing perspective we are trying to position ourselves as a "solution," namely a "scalable, multi-cloud monitoring solution." We've been moving away from "framework" in our messaging, as it sounds work intensive and therefore off putting to enterprises. Let me know if you have further thoughts -- I'm happy to discuss!